app/controllers/friends/base.js

```diff
@@ -22,7 +22,7 @@ export default Ember.Controller.extend({
 
       return false;
     },
-    cancel: function() {
+    cancel() {
```

Comment:
'concise methods' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).